doc/lsst.pipe.base/creating-a-pipeline.rst

```diff
@@ -0,0 +1,448 @@
+path described above. The mapping can optionally contain the keys
+``include``, ``exclude``, and ``importContracts``. The keys ``include`` and
+``exclude`` can be used to specify which labels (or labeled subsets) to
+include or exclude, respectively, when inheriting a ``Pipeline``. The values
+associated with these keys are specified as a yaml list, and these two keys
+are mutually exclusive, only one can be specified at at time. The
+``importContracts`` key is optional and is associated with a boolean value
+that controls wether ``contracts`` from the imported pipeline should be
+included when importing, with a default value of true.
+
+A few further notes about including and excluding. When specifying labels
+with ``include`` or ``exclude``, it is possible to use a labeled subset in
+place of a label. This will have the same effect as typing out all of the
+labels listed in the subset. Another important point is the behavior of
+labels that are not imported (either because they are excluded, or they are
+not part of the include list). If any omitted label appears as part of a
+subset, then the subset subset definition is not imported.
+
+The order that `Pipelines` are listed in the ``imports`` section is not
+important. Another thing to note is that declared labels must be unique
+amongst all inherited `Pipelines`.
+
+Once one or more pipelines is imported the ``PipelineTask``\ s defined in
+the ``tasks`` section are considered. If any new ``labels`` are declared they
+simply extend the total `Pipeline`.
+
+If a ``label`` declared in the the ``tasks`` section was declared in one of
+the imported ``Pipelines``, one of two things happen. If the label is
+associated with the same `PipelineTask` that was declared in the imported 
+pipeline, this definition will be extended. This means that any configs
+declared in the imported `Pipeline` will be merged with configs declared in
+the current `Pipeline` with the current declaration taking config precedence.
+This behavior allows tasks to be extended in the current `Pipeline`.
+
+If the ``label`` declared in the current `Pipeline` is associated with a
+different `PipelineTask` than that of the imported declaration, then the
+label with be considered re-declared and the declaration in the current
+`Pipeline` will be used when processing the complete pipeline. The
+declaration defined in the imported `Pipeline` dropped.
+
+
+-------------------------------------
+obs\_* package overrides for Pipelines
+-------------------------------------
+
+Pipeline files support automatically loading configuration files defined in
+obs packages though a top level key named `instrument`. The value associated
+with this key is a string representing the fully qualified class name of the
+python camera object. For instance for ``obs_subaru`` this would look like:
+
+.. code-block:: yaml
+
+  instrument: lsst.obs.subaru.HyperSuprimeCam
+
+The ``instrument`` is available to all `Pipelines`, but by conventions obs\_*
```

Comment:
"The `instrument` \*key\* is..."

No "s" on "conventions".

"...packages will typically include their own set of customized `Pipeline`s located in a `pipelines/` directory located at the top level of the package.  These often inherit from generic versions defined elsewhere (e.g. the `DRP.yaml` `pipeline` located in the `pipelines/` directory of the `pipe_tasks` repository), and customizations include `PipelineTask` (re)declarations, instrument label, etc."